configure.py

```diff
@@ -200,6 +200,9 @@
     "-str reuse",
     "-multibyte",  # For Wii compilers, replace with `-enc SJIS`
     "-i include",
+    "-i libs/PowerPC_EABI_Support/include",
+    "-i include/dolphin",
+    "-i libs",
     f"-i build/{config.version}/include",
     f"-DBUILD_VERSION={version_num}",
     f"-DVERSION_{config.version}",
```

Comment:
This might need the same fix that you included above.
```suggestion
    f"-DVERSION_={config.version}",
```

Reply:
Nah this is fine, its just the include paths.